libr/asm/p/asm_arm_cs.c

```diff
@@ -98,6 +98,25 @@ static int disassemble(RAsm *a, RAsmOp *op, const ut8 *buf, int len) {
 	}
 
 	n = cs_disasm (cd, buf, R_MIN (4, len), a->pc, 1, &insn);
+
+	if (a->bits == 16) {
+		//patch disassembly if insn in IT block
+		cs_insn *ITinsn = NULL;
+		int ITcounter = 0;
+		int s = cs_disasm (cd, buf-8, len+8, a->pc-8, 5, &ITinsn);
+		for (int i = 0; i < s; i++) {
```

Comment:
same as above. also this code seems copypasta fromt he one above. can you move this to a separate function?